### DISCUSSION TITLE
Embedded Tags Link To Parent Tag

### DISCUSSION THREAD

**Discussion by Zapperz0398** [kctekn/obsidian-TagsRoutes#12] [General]
I see that this plugin supports linking a parent tag to its embedded tag - which is exactly what Obsidian **Does Not** do.

Obsidian puts the graph view with the embedded tag, and presents it as one view:

![Screenshot (8) - edit](https://github.com/user-attachments/assets/dbdf01df-f2bc-4065-9d4f-b12c5265ee1b)

This is not the case with TagRoutes. It actually shows the parent tag, and then descends to its children. It's pretty neat. 

How did you achieve this if I may ask? I ask this because in the README.md, I don't see any mentions of embedded tags.

**Comment by kctekn** [comment]
Simply tag it directly using the format shown in the image, like this:
`#programming/version-control/git`
You can also include multiple tags in one line:
`#programming/version-control/git #tagtopic1 #tagAnotherTopic 2024-08-31 22:20:34`

The plugin will automatically create the hierarchy for you.

**Reply by Zapperz0398** [reply]
Yes, I understand the plugin would do it automatically, but how did YOU do it? Can the same functionality be used in the native Obsidian graph view?

**Reply by kctekn** [reply]
Those changes need to be done in programing level. If we want the native Obsidian graph view to do the same thing, we'll need to change it's source code, while it seems have no other method yet. If you need the tags show in nested way, you can use this plugin as it provides even more features on the tags handle!

**Reply by Zapperz0398** [reply]
I see. Well, thanks for the help. 👍
